src/pages/home/sidebar/SidebarScreen.js

```diff
@@ -29,7 +29,7 @@ import NameValuePair from '../../../libs/actions/NameValuePair';
 
 const propTypes = {
     /* Beta features list */
-    betas: PropTypes.arrayOf(PropTypes.string).isRequired,
+    betas: PropTypes.arrayOf(PropTypes.string),
```

Comment:
This change and the similar one in `WorkspaceCardPage` just kills some distracting console errors.